apply.sh

```diff
@@ -6,7 +6,6 @@ repos=(
 
 args=(
   --allow-added-labels # Keep any existing labels that are not in the config
-  --labels labels.js
```

Comment:
We're now using the default **labels.json**

Newer releases of [`github-label-sync`](https://github.com/Financial-Times/github-label-sync#readme) have added support for [`*.yaml` config files](https://github.com/Financial-Times/github-label-sync/pull/47) (in addition to `*.json`) but the changes meant we lost `*.js` config support via `require()` resolution